spec/unit/rest/route_spec.rb

```diff
@@ -45,10 +44,7 @@
 
       it 'yields the first entry in the server list when server_list is in use' do
         Puppet[:server_list] = [['one.net', 111], ['two.net', 222]]
-        fallback_route = Puppet::Rest::Route.new(api: '/fakeapi/v1/',
-                                                 default_server: nil,
-                                                 default_port: nil,
-                                                 srv_service: nil)
```

Comment:
Lets keep one test where the user would specifcially pass in nil (so we know how things will work w/out the method defaults)

Reply:
I've added a new test that explicitly passes `nil` for the settings.